website/content/api-docs/system/internal-counters.mdx

```diff
@@ -1088,8 +1088,12 @@ months in the requested time range.
 information returned by this API will include activity for this month, however
```

Comment:
should we add something about that -raw flag that is necessary to output the data in the CLI?

Reply:
Ah, yes. That's a great point. I was thinking of adding that separate from the API docs. Possibly here https://developer.hashicorp.com/vault/docs/concepts/client-count.